community/planning/griddle/openapi.yaml

```diff
@@ -0,0 +1,2386 @@
+# Copyright 2021-2022 Cargill Incorporated
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+openapi: 3.0.0
+info:
+  version: 0.4.0
+  title: Grid Daemon REST API
+  description: A REST API providing HTTP/JSON interface to Hyperledger Grid.
+  contact:
+    name: Hyperledger Grid community
+    url: https://grid.hyperledger.org/community/
+  license:
+    name: Apache 2.0
+    url: http://www.apache.org/licenses/LICENSE-2.0
+externalDocs:
+  description: |
+    For more information about how to create Sabre batches, please see the
+    Sabre documentation.
+  url: https://sawtooth.hyperledger.org/docs/sabre/releases/latest/
+paths:
+  # Transactions
+  /statuses:
```

Comment:
This should be `batch_statuses` right ? I think we discussed just using "statuses" but the rest of the endpoints from the dlts (splinter, sawtooth) use `batch_statuses` so it might be a good idea to stay consistent with that

Reply:
Ok, I see we use dashes (and would not support a `batch_statuses` endpoint) this is prob a fine solution for now.